go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/coredns/coredns v1.9.3
 	github.com/davecgh/go-spew v1.1.1
 	github.com/egymgmbh/go-prefix-writer v0.0.0-20180609083313-7326ea162eca
-	github.com/emicklei/go-restful v2.9.5+incompatible
+	github.com/emicklei/go-restful v2.16.0+incompatible
```

Comment:
Revert?

Reply:
Ok, it was coming from other sub-modules, forced all of them down, and this one went down too (including root one too)... :man_juggling: